src/ObjectList.php

```diff
@@ -357,4 +359,19 @@ public function sum(callable $valueProvider, $startValue = null)
             return $total + $valueProvider($item);
         }, $startValue);
     }
+
+    /**
+     * @param callable $filter
+     * @return mixed
+     */
+    public function get(callable $filter)
```

Comment:
I don't like name `get` - more it looks like `filter` or `find`, `search` etc. Get is something that should just return values.